tenzir/functional-test/tests/bloom-filter-context.bats

```diff
@@ -0,0 +1,25 @@
+export TENZIR_PLUGINS="context,enrich"
```

Comment:
@lava or @tobim: how do get this to work?

I'm getting errors in CI:

```
not ok 1 Bloom filter context in 3167ms
# (from function `setup_node' in file /__w/tenzir/tenzir/tenzir/functional-test/tests/../bats-tenzir/src/fixtures.bash, line 22,
#  from function `setup' in test file /__w/tenzir/tenzir/tenzir/functional-test/tests/bloom-filter-context.bats, line 9)
#   `setup_node' failed
# !! invalid_configuration: failed to find the context plugin as libtenzir-plugin-context.so in ["/__w/tenzir/tenzir/build/lib/tenzir/plugins/"]
# /__w/tenzir/tenzir/tenzir/functional-test/tests/../bats-tenzir/src/fixtures.bash: line 26: kill: (16875) - No such process
```

Reply:
You can't write these tests here. The context plugin is closed-source, so you will have to add these tests to the context plugin instead.

I recommend waiting for https://github.com/tenzir/tenzir/pull/3886 to do this, which should very soon be merged.

Reply:
I removed the tests and will migrate them as a follow-up.